### PR TITLE
use a real presubmit

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -6,5 +6,18 @@ aliases:
...
+    - matzew
+    - nachocano
+    - lionelvillard
+    # serving
+    - dprotaso        # API
+    - markusthoemmes  # Scaling
+    - mattmoor        # API
+    - mdemirhan       # Network
+    - tcnghia         # Network
+    - vagababov       # Scaling
```
Comment (mattmoor) [inline OWNERS_ALIASES:23]: Add trailing newline:
```suggestion
    - vagababov       # Scaling

```

**Review by grantr — APPROVED** [review]
/lgtm

**Pull request merged**